politeiawww/cmsuser.go

```diff
@@ -167,7 +171,7 @@ func (p *politeiawww) processRegisterUser(u cms.RegisterUser) (*cms.RegisterUser
 			log.Debugf("RegisterUser failure for %v: user not found",
 				u.Email)
 			return nil, www.UserError{
-				ErrorCode: www.ErrorStatusVerificationTokenInvalid,
+				ErrorCode: www.ErrorStatusCannotCensorComment,
```

Comment:
Hmm not sure what's going on here, but the fact that this is not causing a test to fail means that you're missing a test for this code path.

Reply:
whoops, debug stuff that i hadn't meant to commit. reverting

Reply:
Shouldn't that have caused a test to fail though?

Reply:
Ah yah let me add that test case.

Reply:
done